src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -324,25 +322,20 @@ public void updateFleetStatusUpdateForAllComponents() {
     }
 
     private Boolean deploymentStatusChanged(Map<String, Object> deploymentDetails) {
-        DeploymentType type = Coerce.toEnum(DeploymentType.class, deploymentDetails
-                .get(DEPLOYMENT_TYPE_KEY_NAME));
-        if (type == IOT_JOBS || type == SHADOW) {
-            String status = deploymentDetails.get(DEPLOYMENT_STATUS_KEY_NAME).toString();
-            if (JobStatus.IN_PROGRESS.toString().equals(status)) {
-                isDeploymentInProgress.set(true);
-                return true;
-            }
-            logger.atDebug().log("Updating Fleet Status service for deployment with ID: {}",
-                    deploymentDetails.get(DEPLOYMENT_ID_KEY_NAME));
-            isDeploymentInProgress.set(false);
-            DeploymentInformation deploymentInformation = getDeploymentInformation(deploymentDetails);
-            updateEventTriggeredFleetStatusData(deploymentInformation);
+        String status = deploymentDetails.get(DEPLOYMENT_STATUS_KEY_NAME).toString();
+        if (JobStatus.IN_PROGRESS.toString().equals(status)) {
+            isDeploymentInProgress.set(true);
+            return true;
         }
-        // TODO: [P41214799] Handle local deployment update for FSS
-        return true;
-    }
 
+        logger.atDebug().kv("deployment details", deploymentDetails)
+                .log("Updating Fleet Status service for deployment.");
```

Comment:
_super_ nit: logger appends a period so this will end up with ".." in the log file. We should fix this in the logger so it doesn't append if the string ends with a ".", but in the mean time can you remove that trailing "."

Reply:
Done ...and removed a bunch of other instances.